lib/puppet/provider/mount/parsed.rb

```diff
@@ -123,11 +123,11 @@ def post_parse(result)
         result[:options] = [result[:options],special_options.sort].flatten.compact.join(',')
         if ! result[:device]
           result[:device] = :absent
-          Puppet.err "Prefetch: Mount[#{result[:name]}]: Field 'device' is missing"
+          Puppet.err _("Prefetch: Mount[%{name}]: Field 'device' is missing") % { name: result[:name] }
```

Comment:
Can you add a note that `Prefetch` is a program name here? Same for next externalization